ggplot_Matplotlib_cheatsheet.Rmd

```diff
@@ -0,0 +1,20 @@
+---
+title: "ggplot_Matplotlib_cheatsheet"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+Binghong Yu(by2325)
```

Comment:
once yaml is removed and replaced with a "# Header", the name should go after it (with a newline in between), CUID not necessary

Reply:
> Hi, I just made the change based on your comment. Could you check if this is what you have commented on. Thanks!:)

Reply:
Sorry for the late response! The new first line works, although sorry if I wasn't clear -- "Header" can be whatever you want to name your article, eg: "# Matplotlib cheatsheet" or whatever you would prefer. Also, the other edit you still need to name is renaming your .Rmd file so that it does not use capitalization, meaning changing it from "ggplot_Matplotlib_cheatsheet.Rmd" to "ggplot_matplotlib_cheatsheet.Rmd"

Reply:
Just rename the file！Thanks:)

Reply:
thank you for renaming the file! however, apologies if it wasn't clear in my earlier comment -- your first line should have a "#" and then a whitespace before your page's name, eg: "# Matplotlib cheatsheet" instead of "Header" as it is now. I also think it would be preferable if you removed your CUID from the parentheses next to your name in the 3rd line of your file, although this is probably not required.

Reply:
Does it work now? Please check it now.

Reply:
looks good 👌